.snippets/code/xtokens/ethersjs.md

```diff
@@ -0,0 +1,60 @@
+```javascript
+import abi from './xtokensABI.js'; // Import the xTokens ABI
```

Comment:
maybe we should have a snippet for the ABI too?

Reply:
I've added the ABI as a link

Reply:
perfect thanks!